starter_code/queries.md

```diff
@@ -4,43 +4,158 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+{
+ filter: {
+  name: 'Babelgum'
+ },
+ project: {
+  name: 1
+ }
+}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+{
+ filter: {
+  number_of_employees: {
+   $gt: 5000
+  }
+ },
+ sort: {
+  number_of_employees: -1
+ },
+ limit: 20
+}
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+{
+ filter: {
+  $and: [
```

Comment:
Instead of doing this, a shorter way would be: {founded_year: {$gt : 1999, $lt: 2006}}
Where you have to type founded year just once and filtering it.